ROADMAP.md

```diff
@@ -0,0 +1,32 @@
+# Roadmap
+
+- [Raw devices Implement](#Raw-devices-Implement)
+- [Manage volume group that user created](#Manage-volume-group-that-user-created)
+- [Data Migration and Backup](#Data-Migration-and-Backup)
+- [Nodes and devices selector](#Node-and-devices-selector)
+
+###Raw devices Implement
+
+Not only lvm, operator could orchestrate csi driver that support raw devices. we should implement csi driver.
```

Comment:
If I'm not wrong, this has to implemented in the Topolvm project, right?  If yes, has there been any discussions with the topolvm team about this to see if they are ok with this implementation?

Reply:
It seems that topolvm project didn't make a plan to do that , and I think that raw devices relies on automatic device discovery .  if topolvm project do that , the architecture will change dramatically. So I want to make a separate driver to do that.

Reply:
IMHO, Agree with @sp98, the operator has the management role of topolvm, this feature probably is more on the topolvm side.